### PR TITLE
[#767]: Add "hide page title" control

### PR DESCRIPTION
**Pull request by goldenapples**
Adds a toggle control on the page status box, which toggles whether the page title will be displayed or visually hidden.

image

### CODE REVIEW

File: assets/src/sass/objects/_header.scss [hunk 1]
```diff
@@ -211,6 +211,10 @@ $header-bp-mw: 1049px;
 		z-index: 2;
 		margin-top: rem(30);
 		width: 100%;
+
+		body.hide-page-title & {
```
Comment (MiguelAxcar) [inline assets/src/sass/objects/_header.scss:215]: Hiding the `h1` inside `header-content` containing the page title only may be a better option, as it's currently hiding also an `<h2 class="h4 eyebrow">` which has a back link on some pages.

```suggestion
		body.hide-page-title & h1 {
```

I tested the suggested solution here and it's looking good.

image

**Review by MiguelAxcar — COMMENTED** [review]
That's a very clean and elegant solution, great work @goldenapples ! I dropped a suggestion for your assessment.

**Comment by goldenapples** [comment]
I've made the suggested styling change - nice catch! - and also fixed the meta field permissions so that people who aren't super-admins can edit it. Would you re-review please @MiguelAxcar?

**Review by MiguelAxcar — APPROVED** [review]
Tested locally and it's looking great to me!

**Pull request merged**